delivery-slot-management/README.md

```diff
@@ -0,0 +1,140 @@
+
+![img_13.png](./images/img_13.png)
+
+### Delivery Slot selection
+After the ___home delivery___ mode has been selected, the customer will have to select a delivery slot within the next days to receive its goods.
+![img_14.png](./images/img_14.png)
+
+The minimum and maximum lead time for the home delivery will be configurable through the following properties:
+```
+homeDelivery.minimumLeadTime=1
+homeDelivery.maximumLeadTime=7
+```
+As you can see from the above picture, slots can be Available or Unavailable for customer to select: unavailable slots are those that have exhausted their availability quantity. 
+So for instance if a delivery slot had 3 as available quantity and it has already 3 Delivery Slot Management (aka booking) associated (either in BOOKED or CONFIRMED status), then it would be returned as unavailable for a new customer executing the checkout flow.
+If there are no available slots in between the minimum and maximum lead time an error will be presented to the customer and he'll have to choose a different delivery mode.
+Till the order is not placed, customer can choose to change its delivery slot booking. Once the order will be placed, the delivery slot booking will be placed in CONFIRMED status.
+
+### Booking cancellation (to be implemented)
+A cronjob will run every few minutes to retrieve delivery bookings older then a predefined amount of time configurable thorugh properties (defaul of 30 minutes); if the booking is still in the BOOKED status it will be deleted to free the corresponding delivery slot for some other customer.
+
+### Booking confirmed
+Once the customer place an order with a ___Home Delivery___ shipping mode selected and a Delivery Slot Management (aka booking) associated, during the ___place order___ flow, the Delivery Sloot Management associated will be checked to see if it's still valid and present and then it will be changed to the CONFIRMED status.
+In the Order Confirmation page, the ___Home Delivery___ square box will present the detail of the delivery to the customer
+![img_15.png](./images/img_15.png)
+
+### Customer Support Cockpit - Home Delivery details
+In the Customer Support Cockpit, customer agent will be able to see the details of the Delivery Slot Management associated when browsing a specific customer order with ___Home Delivery___ mode selected
+![img_16.png](./images/img_16.png)
+
+## How to Install
+Here are the instructions to configure locally the extension and make it work. The extension has been developed to work with the standard `electronics-spa accelerator`, although it should easily work with other SAP accelerators as well with small changes.
+
+### Install SAP Commerce Platform
```

Comment:
Link the installation step to the Easy documentation URL: i.e.

https://sap.github.io/easy-extension-framework/getting-started/